infra/modules/worklytics-connector-specs/google-workspace.tf

```diff
@@ -145,6 +145,26 @@ locals {
         "/admin/reports/v1/activity/users/all/applications/meet?maxResults=10"
       ]
       example_api_calls_user_to_impersonate : local.google_workspace_example_admin
+    },
+    "gemini-for-workspace" : {
+      source_kind : "gemini_for_workspace"
```

Comment:
One thing that I've realized with this is the format of the YAML file for rules (and the format of source_kind). Here we are using underscores, meanwhile rest of rules we are using hyphens (see `google-meet`)